### PR TITLE
We want to auto deploy to dev on merges to master.

### PR DESCRIPTION
**Pull request by easternbloc**
This should do just that using the new drone-trigger plugin

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -79,12 +79,16 @@ pipeline:
...
-  # trigger_deploy:
-  #   image: fedora:24
-  #   commands:
-  #     - 'curl -H "Authorization: Bearer ${DRONE_TOKEN}" "https://drone.digital.homeoffice.gov.uk/api/repos/UKHomeOffice/kube-brpapp/builds/1?BRPAPP_IMAGE_TAG=${DRONE_COMMIT_SHA}&NAMESPACE=brp-dev"'
-  #   when:
-  #     branch: master
+  trigger_deploy:
+    image: quay.io/ukhomeofficedigital/drone-trigger:latest
+    drone-server: https://drone.digital.homeoffice.gov.uk
+    repo: ukhomeofficedigital/kube-brpapp
```
Comment (vaijab) [inline .drone.yml]: should be:
`repo: UKHomeOffice/kube-brpapp`

File: .drone.yml [hunk 2]
```diff
@@ -79,12 +79,16 @@ pipeline:
...
+    branch: master
+    deploy-to: dev
+    params:
+      - IMAGE_NAME=ukhomeofficedigital/brpapp:${DRONE_COMMIT_SHA}
```
Comment (vaijab) [inline .drone.yml]: should be:
`- IMAGE_NAME=quay.io/ukhomeofficedigital/brpapp:${DRONE_COMMIT_SHA}`

Comment (easternbloc) [inline .drone.yml]: ah quay is trolling me there oops

**Comment by vaijab** [comment]
LGTM, feel free to merge.

**Pull request merged**